### PR TITLE
Fetch ingredients using api

### PR DESCRIPTION
**Pull request by taranveersingh93**
Description
- Made the fetch function to get ingredients.
- Combined the three fetch API's inside a dataLoad function and on window load, only dataLoad is invoked.

Testing Notes
- App functions as expected.
- All tests still pass

### CODE REVIEW

File: src/apiCalls.js [hunk 1]
```diff
@@ -37,10 +37,23 @@ const fetchRecipes = () => {
         });
 }
 
+const fetchIngredients = () => {
+  fetch(`https://what-s-cookin-starter-kit.herokuapp.com/api/v1/ingredients`)
+  .then(response => response.json())
+  .then(ingredientData => {pageData.allIngredients = ingredientData.ingredients})
+  .catch(error => console.log(error))
```
Comment (mbenfowler) [inline src/apiCalls.js]: these should be indented

File: src/scripts.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import './styles.css'
 import { makeTagActive, closeRecipe, showRecipe, searchForRecipes, switchView } from './domUpdates';
 import { calculateRecipeCost, getIngredientAmounts, getInstructions } from './recipes'; 
 import { ingredientsData } from './data/ingredients';
-import { assignCurrentUser, fetchRecipes } from './apiCalls';
+import { dataLoad, pageData } from './apiCalls';
```
Comment (mbenfowler) [inline src/scripts.js]: looks like `pageData` is no longer needed in scripts?

**Review by mbenfowler — CHANGES_REQUESTED** [review]
damn this is a clean change, just a couple nits

File: src/apiCalls.js [hunk 3]
```diff
@@ -37,10 +37,23 @@ const fetchRecipes = () => {
...
+}
+
+const dataLoad = () => {
+  assignCurrentUser();
+  fetchRecipes();
+  fetchIngredients();
+}
```
Comment (lauraguerra1) [inline src/apiCalls.js:51]: love this!!

File: src/apiCalls.js (hunk 1 above)
Comment (mbenfowler) [inline src/apiCalls.js]: oh and shouldn't it be `console.error(error)`?

**Review by mbenfowler — CHANGES_REQUESTED** [review]

File: src/scripts.js (hunk 2 above)
Comment (lauraguerra1) [reply src/scripts.js]: I am using it in scripts in my branch! Do we mind leaving it there?

**Review by lauraguerra1 — APPROVED** [review]

File: src/apiCalls.js (hunk 1 above)
Comment (taranveersingh93) [reply src/apiCalls.js]: Gosh what would I do without you?

File: src/scripts.js (hunk 2 above)
Comment (taranveersingh93) [reply src/scripts.js]: removed

File: src/apiCalls.js (hunk 1 above)
Comment (taranveersingh93) [reply src/apiCalls.js]: done

**Review by mbenfowler — APPROVED** [review]
wooo

**Comment by mbenfowler** [comment]
[[Iteration 3] Get all ingredients (fetch)](https://trello.com/c/X5kRjEqz/27-iteration-3-get-all-ingredients-fetch)

**Pull request merged**